src/kvirc/ui/KviConsoleWindow.cpp

```diff
@@ -396,7 +393,11 @@ void KviConsoleWindow::saveProperties(KviConfigurationFile * cfg)
 
 void KviConsoleWindow::getBaseLogFileName(QString & buffer)
 {
-	buffer = QString("CONSOLE_%1").arg(context()->id());
```

Comment:
> Sorry, I don't understand what you mean here. Can you give a concrete example of a situation where this might be a problem?

its not so much of a problem as it maybe preferred to know which ID the console is attached to

idk, I dont care either way.

> I don't see how that would affect log naming. What do you mean by "still"?

**still** means I need more coffee... yes youre right, it shouldnt affect logging but it does for channels and queries so Im wondering if this plays nice the same way.
![capture](https://cloud.githubusercontent.com/assets/3521959/16707232/14218a44-45bf-11e6-9bbd-17c0fdc1c79d.PNG)




> Why would you want the ID? IDs are ephemeral, they have no meaning once that console window is closed.

were talking about the case were they arent closed or no?